Edit 'additional resources' on about page

mlinksva opened the pull request:
- add FSF list
- slight rewording, eg OSI list not exactly comprehensive
- add Cowan wizard
- rm TLDRLegal; can't seem to keep junk/spam additions off its home page

mlinksva committed:
Edit 'additional resources' on about page

- add FSF list
- slight rewording, eg OSI list not exactly comprehensive
- add Cowan wizard
- rm TLDRLegal; can't seem to keep junk/spam additions off its home page

waldyrious commented:
Shouldn't the wizards go on the "Tools" section?

waldyrious commented:
By the way, the link for the "License Maven Plugin" in the Tools section needs to be updated to http://www.mojohaus.org/license-maven-plugin/.

mlinksva committed:
rm tools section

first 3 narrow / exper tools
addalicense.com is very relevant but seems unreliable, aim to bring back

mlinksva commented:
'Resources' and 'tools' are both rather generic headings, applicable to just about anything. I just deleted the tools section for now. Thanks for pointing out @waldyrious.
